torchok/models/heads/detection/__init__.py

```diff
@@ -1,6 +1,7 @@
 import importlib
 
 has_mmcv = importlib.util.find_spec("mmcv")
-if has_mmcv is not None:
-    import torchok.models.heads.detection.fcos
-    import torchok.models.heads.detection.detr
+# ToDo: fix mmdet heads
```

Comment:
So, detection doesn't work?

Reply:
Yes, but I am fixing this

Reply:
What does it require to make it work? I'm estimating whether to put it into this PR